spec/controllers/internal_api/v1/pillars_controller_spec.rb

```diff
@@ -527,4 +528,45 @@ def expected_dex_json(num, certificate)
     end
   end
   # rubocop:enable RSpec/ExampleLength
+
+  context "with dex OIDC connectors" do
+    let!(:oidc_connector) { create(:dex_connector_oidc, :skip_validation, name: con_name) }
+    # because name increments every time factory is called, it's not predictable
+    let(:con_name) { "pillar test OIDC connector" }
```

Comment:
This is the same as explained at https://github.com/kubic-project/velum/pull/672#issuecomment-425859643. The factory generates the data, use that in our favor.

Reply:
I was literally just fixing that, having cleaned up the LDAP test. :)

Reply:
Updated in 25a6d12.